TYPO3.Neos/Documentation/CreatingASite/ContentCache.rst

```diff
@@ -164,6 +164,26 @@ In the ``@cache`` meta property the following subproperties are allowed:
 	}
 
 
+When using ``dynamic`` as the cache mode, the cache can be *disabled* by setting the ``entryDiscriminator`` to ``false``.
```

Comment:
Is that `false` or false-ish? The example uses a request parameter, is that a real bool? I doubt it.